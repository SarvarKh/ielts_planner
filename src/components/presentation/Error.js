export default function Error() {
  return (
    <div>
      <h1>Oops! Page not found!</h1>
    </div>
  );
}
