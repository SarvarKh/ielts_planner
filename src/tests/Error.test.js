import React from 'react';
import renderer from 'react-test-renderer';
import Error from '../components/presentation/Error';

it('renders correctly', () => {
  const tree = renderer.create(<Error />).toJSON();
  expect(tree).toMatchSnapshot();
});
