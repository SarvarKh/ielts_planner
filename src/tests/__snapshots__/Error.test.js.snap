// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Oops! Page not found!
  </h1>
</div>
`;
